modules/home/assets/js/app.js

```diff
@@ -1,5 +1,5 @@
 import { DirectionProvider, LocalizationProvider } from '@elementor/ui';
-import { ThemeProvider } from '@elementor/ui/styles';
+import { ThemeProvider } from '@elementor/ui';
```

Comment:
You can add the `ThemeProvider` import to the previous line (both imports are from `@elementor/ui`)

Reply:
@hein-obox Yes the linter failed. LOL it was still WIP.
Updated.

Reply:
@yotamselementor Updated.